src/GitHub.Api/Helpers/RegularExpressions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text.RegularExpressions;
+
+namespace GitHub.Unity.Helpers
+{
+    public static class RegularExpressions
+    {
+        public static readonly Regex BranchNameRegex = new Regex(@"^(?<name>[\.\w\d\/\-_]+)$");
+    }
+
+    public static class BranchNameValidator
+    {
+        public static bool IsBranchNameValid(string branchName)
```

Comment:
Class name feels too specific. `Validation` would probably be good for a class name, then we can dump all the validator helpers in it